backend/hubspot/hubspot.go

```diff
@@ -95,7 +95,7 @@ func getDomain(adminEmail string) (domain string) {
 }
 
 type Api interface {
-	CreateContactForAdmin(ctx context.Context, adminID int, email string, userDefinedRole string, userDefinedPersona string, first string, last string, phone string, referral string) error
```

Comment:
Any reason we removed the string types here? I noticed some of them were still present in the spec that uses HubspotMock adn the implementation below (i.e. `first` and `phone`).

Reply:
in go `first, second string` is just shorthand for `first string, second string`. thought i would consolidate the `string` type to 'group' the variables together, since there are so many (probably would be cleaner to pass a struct around as an argument)

Reply:
That makes sense, and thanks for the learning 🎉